viennacl/ocl/mempool/mempool_utils.hpp

```diff
@@ -0,0 +1,137 @@
+// Various odds and ends
```

Comment:
This shouldn't be necessary. Particularly, we shouldn't add our own error classes (but use VCL's existing ones instead).